### ISSUE TITLE
no-duplicates should ignore flow type imports

### ISSUE THREAD

**Issue by dmnd** [benmosher/eslint-plugin-import#225]
I'm using [flow](http://flowtype.org/) and `no-duplicates` triggers on:

```js
import {someFunction} from '../src/foo';
import type {someType} from '../src/foo';
```

But as far as I know there's no way to import both types and other exports with the same line. `no-duplicates` should consider a flow type import as separate from a module export.

**Comment by benmosher** [comment]
Ah, interesting. So no Lee Byron `export-extensions`-style sytax for Flow yet, then. 😉

I'm inclined to keep it simple and just recommend against using no-duplicates if the project contains flow types, but it seems like a reasonable use case so I would accept a PR that flags multiple type imports or multiple normal imports. 

I'm not sure when I would have a chance to take a stab at this, though.

**Comment by dmnd** [comment]
> I'm not sure when I would have a chance to take a stab at this, though.

Totally not expecting/asking you to do this :+1:

I mainly wanted to open the issue to track it. Who knows, maybe someone will be inspired to send a PR. Maybe that someone will be me if my flow-enabled project gets bigger...
